### PR TITLE
nemo top level folder w ft under

### PR DESCRIPTION
**Pull request by dglogo**

### CODE REVIEW

File: nemo/finetuning/NeMo/README.md
Comment (shubhadeepd) [inline nemo/finetuning/NeMo/README.md:1]: Can we keep just `nemo/finetuning/slm`?
Keeping a directory named NeMo inside nemo looks a bit weird to me.

**Review by shubhadeepd — COMMENTED** [review]
Can we port the nemo evaluator based notebooks here from RAG/notebooks?
https://github.com/NVIDIA/GenerativeAIExamples/tree/dglogo/nemo-folder/RAG/notebooks/nemo/Nemo%20Evaluator%20Llama%203.1%20Workbook

Maybe in `nemo/evaluator`?

File: nemo/finetuning/NeMo/README.md
Comment (dglogo) [reply nemo/finetuning/NeMo/README.md:1]: yup, good call

**Comment by dglogo** [comment]
> Can we port the nemo evaluator based notebooks here from RAG/notebooks? https://github.com/NVIDIA/GenerativeAIExamples/tree/dglogo/nemo-folder/RAG/notebooks/nemo/Nemo%20Evaluator%20Llama%203.1%20Workbook
> 
> Maybe in `nemo/evaluator`?

yup!

**Pull request closed without merging**